src/components/Disclaimer/Disclaimer.tsx

```diff
@@ -2,17 +2,20 @@ import React from 'react';
 import { DisclaimerWrapper } from './Disclaimer.style';
 import { getMetricDisclaimer } from 'common/metric';
 import { Region } from 'common/regions';
+import { MetricToProvenance } from 'components/Disclaimer/utils';
 
 const Disclaimer = ({
   metricName,
   region,
+  provenanceInfo,
 }: {
   metricName: number;
   region: Region;
+  provenanceInfo: MetricToProvenance;
```

Comment:
I wonder if this could actually be just the provenance for `metric` instead of the whole map too?